qubes/app.py

```diff
@@ -1424,6 +1425,20 @@ def register_event_handlers(self, old_connection=None):
                 self._domain_event_callback,
                 None))
 
+        # Stop storage for domains not currently running
```

Comment:
`register_event_handlers` is not a good place for this. If nothing else, then because it is called on reconnection to (possibly restarted) libvirt - even if qubesd hasn't been restarted.
I'd rather prefer another call in `qubesd.py`.